### PR TITLE
ARM slave support

### PR DESCRIPTION
**Pull request by domcleal**
Some fixes from running our slave modules on two new ARM builders.

### CODE REVIEW

File: puppet/modules/utility/manifests/init.pp [hunk 1]
```diff
@@ -28,10 +28,7 @@
 
   package { "ruby-shadow":
     ensure => installed,
-    name => $osfamily ? {
-      Debian => "libshadow-ruby1.8",
```
Comment (mmoll) [inline puppet/modules/utility/manifests/init.pp]: This might still be needed on Ubuntu/precise

**Comment by mmoll** [comment]
one note, :+1: otherwise

File: puppet/modules/utility/manifests/init.pp (hunk 1 above)
Comment (domcleal) [inline puppet/modules/utility/manifests/init.pp]: We've only got Wheezy slaves, and now one Ubuntu 14.10 (which I hope we can rebuild to 16.04 when it's out).

Comment (mmoll) [inline puppet/modules/utility/manifests/init.pp]: perfect then :)

**Comment by domcleal** [comment]
Thanks for the review!

**Pull request merged**